kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -1200,3 +1217,18 @@ execStateReader config st action =
     flip execStateT st
         $ flip runReaderT config
         $ action
+
+checkIfCorrectFilePath :: MonadIO m => FilePath -> m Bool
```

Comment:
The name of this function is vague: what makes a `FilePath` "correct"? Maybe we could call it something like, `doesParentDirectoryExist`.